pkg/secretless/config/v2/config_http_test.go

```diff
@@ -0,0 +1,70 @@
+package v2
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestNewHTTPConfig(t *testing.T) {
+	t.Run("valid config with 'authenticateURLsMatching' string", func(t *testing.T) {
```

Comment:
The test names in this file don't really explain what you're testing (is `valid config with 'authenticateURLsMatching' string` good? is it bad? does it return an error?)